### PR TITLE
i18n(ko-KR): translate `authoring-content.md`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

translate `authoring-content.md`

### CODE REVIEW

File: docs/src/content/docs/ko/guides/authoring-content.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+Starlight는 `.md` 파일에서 제목 및 설명과 같은 메타데이터를 정의하기 위해 [Markdown](https://daringfireball.net/projects/markdown/)의 모든 구문과 frontmatter [YAML](https://dev.to/paulasantamaria/introduction-to-yaml-125f)을 지원합니다.
+
+해당 파일 형식을 사용하는 경우 Markdown 지원 및 사용법이 다를 수 있으므로 [MDX 문서](https://mdxjs.com/docs/what-is-mdx/#markdown) 또는 [Markdoc 문서](https://markdoc.dev/docs/syntax)를 확인하세요.
+
+## 인라인 스타일
+
+텍스트는 **굵게**, _기울임꼴_ 또는 ~취소선~으로 표시할 수 있습니다.
+
+```md
+텍스트는 **굵게**, _기울임꼴_ 또는 ~취소선~으로 표시할 수 있습니다.
```
Comment (delucis) [inline docs/src/content/docs/ko/guides/authoring-content.md:15]: I think we lost some of the `~` here to indicate the strikethrough behaviour:

```suggestion
텍스트는 **굵게**, _기울임꼴_ 또는 ~~취소선~으로 표시할 수 있습니다~~.

```md
텍스트는 **굵게**, _기울임꼴_ 또는 ~~취소선~으로 표시할 수 있습니다~~.
```

File: docs/src/content/docs/ko/guides/authoring-content.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+백틱을 사용하여 `인라인 코드`를 강조 표시할 수 있습니다.
+
+```md
+백틱을 사용하여 `인라인 코드`를 강조 표시할 수 있습니다.
+```
+
+## 이미지
+
+Starlight의 이미지는 [Astro에 내장된 최적화된 자산 지원](https://docs.astro.build/en/guides/assets/)을 사용합니다.
```
Comment (delucis) [inline docs/src/content/docs/ko/guides/authoring-content.md:32]: Although it’s not translated yet, it’s probably better to link to the `/ko/` version of this page in the Astro docs:

```suggestion
Starlight의 이미지는 [Astro에 내장된 최적화된 자산 지원](https://docs.astro.build/ko/guides/assets/)을 사용합니다.
```

File: docs/src/content/docs/ko/guides/authoring-content.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+주석 유형 다음에 대괄호를 사용해 주석의 제목을 지정할 수 있습니다. `:::tip[알고 계셨나요?]`
+
+:::tip[알고 계셨나요?]
+
+Astro는 ["Islands Architecture"](https://docs.astro.build/en/concepts/islands/)를 사용하여 더 빠른 웹사이트를 구축할 수 있도록 도와줍니다.
+:::
+
+```md
+:::tip[알고 계셨나요?]
+Astro는 ["Islands Architecture"](https://docs.astro.build/en/concepts/islands/)를 사용하여 더 빠른 웹사이트를 구축할 수 있도록 도와줍니다.
```
Comment (delucis) [inline docs/src/content/docs/ko/guides/authoring-content.md:134]: Same here, we can use the Korean URL:

````suggestion
Astro는 ["Islands Architecture"](https://docs.astro.build/ko/concepts/islands/)를 사용하여 더 빠른 웹사이트를 구축할 수 있도록 도와줍니다.
:::

```md
:::tip[알고 계셨나요?]
Astro는 ["Islands Architecture"](https://docs.astro.build/ko/concepts/islands/)를 사용하여 더 빠른 웹사이트를 구축할 수 있도록 도와줍니다.
````

**Review by delucis — COMMENTED** [review]
Thank you for the translation @jsparkdev! I spotted a couple of small things we could improve.

**Comment by jsparkdev** [comment]
There are a lot of mistakes in this translation.

Thanks for the nice code review!

**Pull request merged**